parseURL.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "github.com/xmidt-org/bascule/basculehttp"
+
+type parseURLOption struct {
```

Comment:
Since usually we call functions used to configure structs `Option`s, I'm not sure if it makes sense to use that word here? My suggestion is to use `Config` instead, or add `fx.In` and call it `parseURLIn` instead (not sure if this works?). 🤷 I don't feel that strongly about this.
```suggestion
type parseURLConfig struct {
```

Reply:
Yeah, cool. I'll add the `Config` suffix 👍